terraform/aws/terraform.tfvars.json

```diff
@@ -26,6 +28,7 @@
                 "arn:aws:iam::653761790766:role/operator",
                 "arn:aws:iam::653761790766:role/breakglass",
                 "arn:aws:iam::313879017102:role/production-app-task-role",
+                "arn:aws:iam::313879017102:role/event-received-production",
```

Comment:
I think this should be your demo account, not production?

Reply:
demo is in the dev account and is covered by the root below